src/api.ts

```diff
@@ -482,10 +483,21 @@ const tokenEndpoint = async (req: FastifyRequest, reply: FastifyReply) => {
     } catch (e) {
         const error = e as TokenError
         console.error('token endpoint fault',error)
+        setSessionCookie(reply, '')
         return reply.code(error.statusCode || 500).send({error: 'token parsing'})
     }
 }
 
+const logoutEndpoint = async (req: FastifyRequest, reply: FastifyReply) => {
```

Comment:
This logout endpoint doesn't hit the hello logout piece - we'll need to continue doing that on the front end until/unless Dick has a better solution for this.